### PR TITLE
added installation via helm section

### PR DESCRIPTION
**Pull request by jesse-codefresh**

### CODE REVIEW

**Comment by itai-codefresh** [comment]
[CI] Staging environment is at http://staging-docs.codefresh.io.s3-website.us-east-1.amazonaws.com/helm-installation/docs/

**Comment by itai-codefresh** [comment]
[CI] Staging environment is at http://staging-docs.codefresh.io.s3-website.us-east-1.amazonaws.com/helm-installation/docs/

**Comment by itai-codefresh** [comment]
[CI] Staging environment is at http://staging-docs.codefresh.io.s3-website.us-east-1.amazonaws.com/helm-installation/docs/

**Comment by itai-codefresh** [comment]
[CI] Staging environment is at http://staging-docs.codefresh.io.s3-website.us-east-1.amazonaws.com/helm-installation/docs/

File: _docs/administration/codefresh-runner.md [hunk 1]
```diff
@@ -140,6 +140,31 @@ used by the installer will be instead saved locally in a folder `./codefresh_man
 
 You can then inspect the manifests locally or edit/apply them manually if you wish.
 
+## Installing Codefresh Runner with Helm
+
+To install the [Codefresh Runner](https://codefresh.io/docs/docs/administration/codefresh-runner/) using helm you, follow these steps:
```
Comment (kostis-codefresh) [inline _docs/administration/codefresh-runner.md:145]: Remove the link from here. It is mentioning the page itself.

File: _docs/administration/codefresh-runner.md [hunk 2]
```diff
@@ -140,6 +140,31 @@ used by the installer will be instead saved locally in a folder `./codefresh_man
...
+
+1. Download the Codefresh CLI and authenticate it with your Codefresh account. Click [here](https://codefresh-io.github.io/cli/getting-started/) for more detailed instructions.
+2. Run the following command to create all of the necessary enitites in Codefresh:
```
Comment (kostis-codefresh) [inline _docs/administration/codefresh-runner.md:148]: I think it should be **entities**

**Comment by itai-codefresh** [comment]
[CI] Staging environment is at http://staging-docs.codefresh.io.s3-website.us-east-1.amazonaws.com/helm-installation/docs/

**Pull request merged**